prov/shm/src/smr_rma.c

```diff
@@ -171,12 +164,17 @@ static ssize_t smr_generic_rma(struct smr_ep *ep, const struct iovec *iov,
 	proto = smr_select_proto(use_ipc, smr_cma_enabled(ep, peer_smr), iface,
 				 op, total_len, op_flags);
 
+	cmd = smr_get_cmd(peer_smr, &pos);
```

Comment:
Ideally, the caller of get_cmd I think shouldn't have to track its position. Can we modify the code so the position is tracked internally and the user just has to keep track of the cmd entry pointer itself?